### PR TITLE
acronyms and suggestions

### PR DESCRIPTION
**Pull request by higs4281**
updates the list; yes i'd rather have a life.

### CODE REVIEW

File: acro.json [hunk 1]
```diff
@@ -356,6 +377,13 @@
         "acro": "CTT", 
         "name": "compensatory time for travel"
     }, 
+    "DOM": {
+        "note": "", 
+        "link_label": "DOE",
```
Comment (Scotchester) [inline acro.json]: `link_label` incorrect/not needed?

**Comment by higs4281** [comment]
a most excellent catch @Scotchester

File: acro.json [hunk 2]
```diff
@@ -2209,19 +2272,12 @@
...
-    }, 
-    "FFIEC": {
-        "note": "an interagency council", 
-        "link_label": "website", 
-        "link": "https://www.ffiec.gov/", 
-        "acro": "FFIEC", 
-        "name": "Federal Financial Institutions Examination Council"
+    "COPPA": {
+        "note": "requires parental consent when user testing involves children 13 and younger", 
+        "link_label": "web pageCOR",
```
Comment (Scotchester) [inline acro.json]: Stray `COR` here, and it looks like the COR entry got lost completely.

**Comment by Scotchester** [comment]
Is there any particular rhyme or reason to some of the reordering being done in this PR? (It'd be nice to have all the entries alphabetized, at some point.)

File: acro.json (hunk 2 above)
Comment (higs4281) [inline acro.json]: arrrr. sorry, scott.
my alphabetized list was getting un-alphaed by jsonifying a python (unordered) dict. i switched to ordered dict, which will be sane now, but which replaces the whole shebang this time.

**Comment by higs4281** [comment]
sorry, scott.
my alphabetized list was getting un-alphaed by jsonifying a python (unordered) dict. i switched to ordered dict, which will be sane now, but which replaces the whole shebang this time.

**Comment by Scotchester** [comment]
Ahhhh, gotcha :) We'll it won't show me the diff anymore, but I'll trust that it's good ;)

**Pull request merged**